zephyr/lvgl.c

```diff
@@ -29,8 +29,17 @@ static lv_indev_drv_t indev_drv;
 
 static lv_disp_draw_buf_t disp_buf;
 
+#if !DT_HAS_CHOSEN(zephyr_display)
+#error "LVGL requires a valid /chosen/zephyr,display device tree property"
+#endif
+
+#define DISPLAY_NODE DT_CHOSEN(zephyr_display)
+
+#define LVGL_Z_HOR_RES DT_PROP(DISPLAY_NODE, width)
+#define LVGL_Z_VER_RES DT_PROP(DISPLAY_NODE, height)
```

Comment:
why not just `HOR_RES`? `LVGL_Z`/`LV_Z` prefix is used for Zephyr specific Kconfig settings

Reply:
Renaming to DISPLAY_(WIDTH|HEIGHT).